sdk/webpubsub/Azure.Messaging.WebPubSub.Client/tests/Azure.Messaging.WebPubSub.Client.Tests.csproj

```diff
@@ -12,7 +11,7 @@
     <PackageReference Include="Azure.Messaging.WebPubSub" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(TargetFramework) != 'net461'">
```

Comment:
@zackliu this change could also potentially account for the test failures only on net461. If there is something in that this library depends on that doesn't work on net461.